test/Microsoft.ML.TestFramework/TestCommandBase.cs

```diff
@@ -480,7 +480,7 @@ private string DataArg(string dataPath)
 
         protected void TestPipeFromModel(string dataPath, OutputPath model)
         {
-            using (var env = new TlcEnvironment(new SysRandom(42)))
+            using (var env = new ConsoleEnvironment(42))
```

Comment:
This isn't an equivalent change, right?  It used to use `SysRandom`, but now it will use `TauswortheHybrid`.